bigtop-packages/src/common/hbase/do-component-build

```diff
@@ -36,13 +36,20 @@ if [ $HOSTTYPE = "aarch64" ] ; then
    sed -i '/<version>1.5.0-alpha.6<\/version>/,+1d' ./pom.xml
    sed -i '/<artifactId>asciidoctorj-pdf<\/artifactId>/a\<version>1.5.0-alpha.6<\/version>\n</dependency>\n<dependency>\n<groupId>org.jruby<\/groupId>\n<artifactId>jruby-complete<\/artifactId>\n<version>9.1.8.0<\/version>\n<\/dependency>' ./pom.xml
 fi
-mvn -DskipTests -Dslf4j.version=1.6.1         \
-    -Dhadoop-two.version=$HADOOP_VERSION      \
-    -Dzookeeper.version=$ZOOKEEPER_VERSION    \
-    -Dcheckstyle.skip=true                    \
-    ${MAVEN_ADDITIONAL} \
-    install site assembly:single "$@"
+
+MAVEN_PARAMS=" -DskipTests "
+MAVEN_PARAMS+=" -Dslf4j.version=1.7.25 "
+MAVEN_PARAMS+=" -Dhadoop-two.version=${HADOOP_VERSION} "
+MAVEN_PARAMS+=" -Dzookeeper.version=${ZOOKEEPER_VERSION} "
+MAVEN_PARAMS+=" -Dcheckstyle.skip=true "
+MAVEN_PARAMS+=" -Dmaven.repo.local=${HOME}/.m2/repository "
```

Comment:
I wonder why introduce this line? It seems to be identical to default path.

Reply:
I used the line for my try and error. I think we can remove it now. updating the PR.